install-ml.py

```diff
@@ -4,9 +4,6 @@
 
 version = "".join([str(v) for v in sys.version_info[:2]])
 if version == "27":
-    call(["pip", "install", "http://download.pytorch.org/whl/cpu/torch-0.4.1-cp27-cp27mu-linux_x86_64.whl"])
-    call(["pip", "install", "torchvision"])
-else:
-    call(["pip", "install", "torch_nightly", "-f",
-          "https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html"])
-    call(["pip", "install", "torchvision-nightly"])
+    pass
+    #call(["pip", "install", "https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-1.12.0-cp27-none-linux_x86_64.whl"])
```

Comment:
Not using this file anymore, but keeping it around incase it's complicated to install ml shit in the future.

Reply:
Add a comment that this is no longer used in circle-ci (assuming that was its only purpose)